### DISCUSSION TITLE
Develop an object detection model for Artpec8

### DISCUSSION THREAD

**Discussion by MenahemOwlytics** [AxisCommunications/acap-computer-vision-sdk-examples#169] [Q&A] [answered]
My task is to build a model for Artpec8 chips that detects specific objects

After many unsuccessful attempts to convert existing models, I decided to build the model from scratch based on [examples/tree/addbf72c3ee5d35ca9fecf7c6a59598d1a54f6b8/tensorflow-to-larod-artpec8](https://github.com/AxisCommunications/acap-native-sdk-examples/tree/addbf72c3ee5d35ca9fecf7c6a59598d1a54f6b8/tensorflow-to-larod-artpec8)

This project is for object classification, not object recognition. 

Is there anyone who has built a model of object recognition in Tensorflow 1 from scratch?

Is there any source code for building this working TFlite model?

[https://github.com/google-coral/edgetpu/raw/master/test_data/ssd_mobilenet_v2_coco_quant_postprocess.tflite](https://github.com/google-coral/edgetpu/raw/master/test_data/ssd_mobilenet_v2_coco_quant_postprocess.tflite)

Thanks

**Comment by Corallo** [comment, marked as answer]
Hi @MenahemOwlytics

I was a bit confused by the name you used.
The example we have is for _image_ classification. 
For Object _detection_ we don't have a guide to train, but we can recommend using the[ tensorflow object detector API](https://github.com/tensorflow/models/tree/master/research/object_detection).

See here:
https://github.com/google-coral/tutorials/blob/master/retrain_ssdlite_mobiledet_qat_tf1.ipynb
and to convert to tflite:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/r1/convert/cmdline_examples.md
